README.md

```diff
@@ -98,10 +102,13 @@ Follow these steps to get started:
 
 * Pick the right base image for your application stack.
   We publish the following distroless base images on `gcr.io`:
+    * [gcr.io/distroless/static-debian12](base/README.md)
     * [gcr.io/distroless/static-debian11](base/README.md)
+    * [gcr.io/distroless/base-debian12](base/README.md)
     * [gcr.io/distroless/base-debian11](base/README.md)
```

Comment:
Sorry, this appears to be a previous oversight from the maintainers side, but since you're in here can we add in base-nossl here as well?